mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -14,3 +14,19 @@ export const shouldEnableExperimentDatasetTracking = () => true;
  * UI feature preview: enables artifact-based ML experiment output data analysis, used for evaluating LLM prediction data
  */
 export const shouldEnableArtifactBasedEvaluation = () => true;
+
+export const shouldEnableDatasetsDropdown = () => false;
+
+/**
+ * UI feature preview: enables prompt lab
+ */
+export const shouldEnablePromptLab = () => true;
+
+export const shouldUsePathRouting = () => false;
+
+/**
+ * A flag determining if we should display "New model registry UI" toggle switch.
+ */
+export const shouldUseToggleModelsNextUI = () => {
+  return true;
```

Comment:
@harupy can we set this value to `false`? That way, the aliases UI will not be displayed in the release. It isn't ready yet. Also, happy to make this change in a follow-up PR, where I validate it there.